examples/disconnect_client.py

```diff
@@ -0,0 +1,26 @@
+"""This example shows how to clear resources owned by a client and how to disconnect a client from scheduler."""
+
+from scaler import Client
+from scaler.client.client import Client
+from scaler.cluster.combo import SchedulerClusterCombo
+
+
+def main():
+    cluster = SchedulerClusterCombo(n_workers=10)
+    client = Client(address=cluster.get_address())
+    # Client.clear will clear all computation resources owned by the client. All unfinished tasks will be cancelled,
+    # and all object reference will be invalidated. The client can reinstall tasks as it wishes.
```

Comment:
I think we should use "submit" here and in similar cases.

```suggestion
    # and all object reference will be invalidated. The client can submit tasks as it wishes.
```